ncpi_fhir_utility/update_versions.py

```diff
@@ -14,15 +15,14 @@
 
 
 def bump_ig_version(filePath, new_version):
-    with filePath.open(mode="r") as f:
-        body = [
-            line
-            if not line.startswith("version ")
-            else f"version = {new_version}\n"
-            for line in f
-        ]
-    with filePath.open(mode="w") as f:
-        f.writelines(body)
+    config = configparser.ConfigParser()
+    if config.read(filePath) and "IG" in config:
+        if "version" in config["IG"]:
```

Comment:
If a version is not in the `ini` it probably should be?

Reply:
I think same goes with `date` in the resource payload. If it's not there, we should probably add it.

Reply:
no idea! but we can certainly do that